abr_control/arms/jaco2/config.py

```diff
@@ -55,6 +55,10 @@ def __init__(self, hand_attached=False, **kwargs):
 
         self.REST_ANGLES = np.array([None, 2.42, 2.42, 0.0, 0.0, 0.0])
 
+        # position to move to before switching to torque mode
+        self.START_ANGLES = np.array(
```

Comment:
why are we adding starting angles?

Reply:
I thought we agreed to add this to each arm? I need it for setting a start position

Reply:
hmm, right. does VREP use these start angles too?

Reply:
also, can you add start angles to the one, two, and three joint configs too?